app/views/users/mfa_selection/index.html.erb

```diff
@@ -4,6 +4,21 @@
 
 <p class="maxw-mobile-lg margin-bottom-0"><%= @presenter.intro %></p>
 
+<h2 class="margin-top-2 margin-bottom-1">
+  <%= t('headings.account.two_factor') %>
+</h2>
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <ul class="usa-icon-list">
+      <% @presenter.options.each do |option| %>
+        <% if option.mfa_configuration_count > 0 %>
+          <%= render partial: 'partials/multi_factor_authentication/selected_mfa_option', locals: { option: option } %>
+        <% end %>
+      <% end %>
+    </ul>
+  </div>
```

Comment:
I think we're missing a closing `</div>` tag here, and it's breaking the bottom of the page:

![image](https://github.com/18F/identity-idp/assets/1779930/3ed1f246-ffec-4f6d-9ff1-e9a3d4b2b768)

```suggestion
  </div>
</div>
```